src/Update.hs

```diff
@@ -607,6 +607,9 @@ prMessage updateEnv isBroken metaDescription metaHomepage metaChangelog rewriteM
 
        $maintainersCc
 
+       > [!TIP]
+       > As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** after approving this update to automatically merge this PR using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).
```

Comment:
> after approving this update

Does this mean approving with github "review changes"?

Reply:
The bot itself doesn't really care if the PR is approved or not in GitHub, my reasoning was if a maintainer approves the changes made in the PR, then they probably tested it already, and if so, should merge.

Reply:
Would **after *testing* this update** be more appropriate?

Reply:
Sorry, I missed this.

I'd prefer to be as concise as possible as PR message content is already rather long.